packages/ui-react/src/components/Payment/Payment.tsx

```diff
@@ -203,22 +211,31 @@ const Payment = ({
                   )}
                 </div>
               )}
-              {showChangeSubscriptionButton && (
-                <Button
-                  className={styles.upgradeSubscription}
-                  label={t('user:payment.change_subscription')}
-                  disabled={!canRenewSubscription && activeSubscription.status === 'cancelled'}
-                  onClick={() => {
-                    if (offers.length > 1 && !canRenewSubscription) {
-                      setIsChangingOffer(true);
-                    } else {
-                      onUpgradeSubscriptionClick?.();
-                    }
-                  }}
-                  fullWidth={isMobile}
-                  color="primary"
-                  data-testid="change-subscription-button"
-                />
+              {isExternalPaymentProvider ? (
+                <p className={styles.explanation}>
+                  {t('account:external_payment.explanation', { paymentProvider })}{' '}
+                  <Link href={paymentProviderLink} target="_blank">
```

Comment:
Maybe add `rel="noopener noreferrer"` for safety?

Reply:
This comment is outdated. Since 2021 it is not needed. `noreferrer` is optional

Reply:
The Link component does that automagically when the `target` is set to `_blank` 😄